gateway-service-impl/build.gradle.kts

```diff
@@ -21,16 +21,16 @@ dependencies {
   implementation("org.hypertrace.entity.service:entity-service-client:0.8.56")
   implementation("org.hypertrace.entity.service:entity-service-api:0.8.56")
   implementation("org.hypertrace.core.grpcutils:grpc-context-utils:0.11.2")
-  implementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.11.2")
-  implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.49")
+  implementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.12.0")
```

Comment:
context utils right above was not upgraded. server utils, grpc etc. in the next file were not upgraded. Make sure to update all versions consistently.